examples/ota-requestor-app/ota-requestor-common/BDXDownloader.cpp

```diff
@@ -107,11 +107,17 @@ void BdxDownloader::HandleTransferSessionOutput(TransferSession::OutputEvent & e
     case TransferSession::OutputEventType::kTransferTimeout:
         ChipLogError(BDX, "Transfer timed out");
         mTransfer.Reset();
+        mExchangeCtx->Close();
         break;
+    case TransferSession::OutputEventType::kAckEOFReceived:
```

Comment:
A BDX Receiver (Downloader) will never receive a `BlockAckEOF` message. It will only receive `BlockEOF` and then _send_ `BlockAckEOF`. So this can be left as it was with the other unexpected event types.